runtime/CMakeLists.txt

```diff
@@ -18,6 +19,47 @@ add_custom_command(OUTPUT ${GENERATED_SYN_VIM}
     ${PROJECT_SOURCE_DIR}/src/nvim/eval.c
 )
 
+if(POLICY CMP0054)
+  cmake_policy(SET CMP0054 OLD)
+endif()
+
+file(GLOB PACKAGES ${PROJECT_SOURCE_DIR}/runtime/pack/dist/opt/*)
+
+set(GENERATED_PACKAGE_TAGS)
+foreach(PACKAGE ${PACKAGES})
```

Comment:
I'm sure much of this could be done better by someone more experienced with CMake.